deploy/openj9-chart/Chart.yaml

```diff
@@ -0,0 +1,32 @@
+###############################################################################
+# Copyright (c) 2019 IBM Corp.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+apiVersion: v1
+name: openj9-chart
+version: 0.1.0
+kubeVersion: ">=1.11.0"
+description: OpenJ9 helm chart
+keywords:
+  - amd64
+  - ppc64le
+  - RHOCP
+  - runtime
+  - OpenJ9
+sources:
+  - https://github.com/eclipse/openj9
+maintainers:
+  - name: Java Acceleration
```

Comment:
Please state a name we can agree and use it instead.

Reply:
Removed `maintainers` in latest commit since this field is optional. Should we still keep a maintainer for the chart? If so, I could put down the names of our team or please suggest if there is a better candidate.